projects/jpeg-d/helper/index.js

```diff
@@ -18,11 +18,13 @@ const CRYPTO_PUNK_VAULT = "0xD636a2fC1C18A54dB4442c3249D5e620cf8fE98F";
 const ETHER_ROCKS_VAULT = "0x6837a113aa7393ffbd5f7464e7313593cd2dd560";
 const BAYC_VAULT = "0x271c7603aaf2bd8f68e8ca60f4a4f22c4920259f";
 const MAYC_VAULT = "0x7b179f9bfbe50cfa401c1cdde3cb2c339c6635f3";
+const DOODLES_VAULT = "0x0a36f4bf39ed7d4718bd1b8dd759c19986ccd1a7";
 
 const NFT_CHAINLINKED_ARRAY = [
     CRYPTO_PUNK_VAULT,
     BAYC_VAULT,
-    MAYC_VAULT
+    MAYC_VAULT,
+    DOODLES_VAULT
```

Comment:
is there some way to fetch this vault list on chain? maybe a log is generated each time there is a new vault?